src/web/mjs/connectors/VizShonenJump.mjs

```diff
@@ -43,17 +57,29 @@ export default class VizShonenJump extends Connector {
             return;
         }
 
+        data = [...data.querySelectorAll('a.o_chapters-link')];
+
         data = [...data.querySelectorAll('div.o_sort_container div.o_sortable a.o_chapters-link')];
 
         return data.map(manga => {
             return {
                 id: manga.pathname,
-                title: manga.querySelector(':nth-child(2)').innerText.trim()
+                title: manga.innerText.trim()
             };
         });
     }
 
-    async _getChapters(manga) {
+    async _getMangas() {
+        const mangaAvailibleByVolumes = await this._getMangasAvailibleByVolumes();
+        const mangaAvailibleByChapters = await this._getMangasAvailibleByChapters();
+        let both = mangaAvailibleByVolumes.concat(mangaAvailibleByChapters);
```

Comment:
May be simplified with `spread` operator.
Also the sort by title is not required if i remember correctly as this is already done during further processing of the manga list.
```javascript
return [ ... mangaAvailibleByVolumes, ... mangaAvailibleByChapters ];
```